lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-infra_manager-template.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Openstack_InfraManager_Template < MiqAeServiceTemplateInfra
```

Comment:
This should follow the same sub-classing as the active_record model.  The parent class needs to be 
`MiqAeServiceManageIQ_Providers_InfraManager_Template`